src/views/Timeline.vue

```diff
@@ -124,6 +132,22 @@ export default {
 			// reset component
 			this.resetState()
 
+			// content is completely different
+			this.$emit('update:loading', true)
+			this.fetchContent()
+		},
+		async onlyPhotos() {
```

Comment:
You can't do this, it will create issues as `this.onlyPhotos` is both a method and a prop ?

Reply:
Ah, sorry I misread. I thought this was in methods, not in watch! All good! :100: